keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#include "ergodox.h"
+// #include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+#include "eeconfig.h"
+#include "eeprom.h"
+#include "keymap_colemak.h"
+
+extern keymap_config_t keymap_config;
+
+enum ergodox_layers {
+  _QWERTY,
+  _COLEMAK,
+  _QWOC,
+  _LOWER,
+  _RAISE,
+  _PLOVER,
+// Intermediate layers for SuperDuper (Combo keys does not work on Infinity yet)
+  _SUPER,
+  _DUPER,
+  _SUPERDUPER,
+  _MOUSE,
+  _ADJUST,
+  _MDIA,
+  _SYMB,
+};
+
+enum ergodox_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  QWOC,
+  PLOVER,
+  SUPER,
+  DUPER,
+  SUPERDUPER,
+  MOUSE,
+  LOWER,
+  RAISE,
+  BACKLIT,
+  EXT_PLV,
+  SDTOGG, // Toggle SuperDuper
+  EPRM,
+  VRSN,
+  RGB_SLD
+};
+
+enum functions {
```

Comment:
Could you actually remove this? 

We have deprecated macros/functions in favor of `process_record_user`.